modules/Miss.js

```diff
@@ -40,12 +40,18 @@ class Miss extends React.Component {
     return (
       <LocationSubscriber>
         {(location) => {
-          const { render, component:Component } = this.props
+          const { children, render, component:Component } = this.props
           const { noMatchesInContext } = this.state
           const { serverRouter, match } = this.context
           const noMatchesOnServerContext = serverRouter &&
             serverRouter.missedAtIndex(match.serverRouterIndex)
-          if (noMatchesInContext || noMatchesOnServerContext) {
+          const matched = !(noMatchesInContext || noMatchesOnServerContext)
```

Comment:
(see the example [here](https://react-router.now.sh/Match) under "children: func"